### ISSUE TITLE
Obsolete GO Terms present in the noctua-gpad products/annotations 'with' field

### ISSUE THREAD

**Issue by sierra-moxon** [geneontology/go-site#2110]
For example:
http://snapshot.geneontology.org/annotations/zfin.gpad.gz 
```
ZFIN    ZDB-GENE-980526-362    acts_upstream_of_or_within GO:0007155 ZFIN:ZDB-PUB-041022-3|PMID:15494018    ECO:0000305    GO:0005913    20041026   ZFIN      contributor=GOC:zfin_curators|model-state=production|noctua-model-id=gomodel:ZFIN_ZDB-GENE-980526-362
```

**Comment by pgaudet** [comment]
I think this is done - needs testing

**Comment by pgaudet** [comment]
Done in https://github.com/geneontology/go-site/issues/793

**Comment by kltm** [comment]
Talking to @pgaudet , we believe this is now fixed w/GORULE:0000020.
